test/test_utils/sagemaker.py

```diff
@@ -392,9 +407,8 @@ def execute_local_tests(image, pytest_cache_params):
         if ec2_client and ec2_key_name:
             print(f"Destroying ssh Key_pair for image: {image}")
             destroy_ssh_keypair(ec2_client, ec2_key_name)
-        # return None here to prevent errors from multiprocessing.map(). Without this it returns some object by default
-        # which is causing "cannot pickle '_thread.lock' object" error
-        return None
+
+    return test_success
```

Comment:
Should this not be in the finally code block? Do we want to return a failure for an exception raised during testing?

Reply:
No, the try, except, and finally blocks do not need to return.

Right now, any exceptions raised will be printed out to console, and the list of images that failed tests will be printed out after all parallel processes are joined.